app/javascript/pages/MyEvents/index.js

```diff
@@ -146,7 +146,7 @@ const renderField = props => {
 
 const CreateEditDialog = ({ offices, eventTypes, organizations, onCancel, popover, onSubmit }) => {
   const event = popover.data
-  const isNewEvent = R.isNil(event)
+  const isNewEvent = R.isNil(event) || (event && event.isNew)
   return (
     <Dialog
       title={isNewEvent ? 'Create Event' : 'Edit Event'}
```

Comment:
2. make this `event.isNew`

Reply:
`isNewEvent` is called down below, so we can just assign `isNewEvent = event.isNew` above.